t/202-references.t

```diff
@@ -35,18 +35,18 @@ subtest "URL handling" => {
 }
 
 subtest "leading whitespace references" => {
-    my $reference = Pod::FormattingCode.new(
-        type     => 'X',
-        meta => [["meta", " meta1"]],
-    );
-    my @references = Documentable::Index.new(
-        pod    => $reference,
-        origin => Documentable::Primary,
-        meta   => [["meta", " meta1"]]
-    );
-    for @references -> $ref {
-        is $ref.name.starts-with(" "), False, "leading whitespace";
-    }
+   my $reference = Pod::FormattingCode.new(
```

Comment:
Is this changed indentation to 3? Any reason?

Reply:
Just a mistake, fixed.